core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala

```diff
@@ -223,7 +223,7 @@ class AuthorizerIntegrationTest extends BaseRequestTest {
       .find(x => x.topicName == tp.topic).get.partitions.asScala
       .find(p => p.partitionIndex == tp.partition).get.errorCode)),
     ApiKeys.DESCRIBE_LOG_DIRS -> ((resp: DescribeLogDirsResponse) =>
-      if (resp.data.results.size() > 0) Errors.forCode(resp.data.results.get(0).errorCode) else Errors.CLUSTER_AUTHORIZATION_FAILED),
+      Errors.forCode(if (resp.data.results.size() > 0) resp.data.results.get(0).errorCode() else resp.data.errorCode())),
```

Comment:
nit: I would remove the parenthesis after `errorCode` to stay consistent with the surrounding code style.